lib/tasks/valid_test_data_generator.rb

```diff
@@ -45,27 +45,33 @@ def generate_new_participants(school:, count:, sparsity_uplift:, pupil_premium_u
       while count.positive?
```

Comment:
It can be done in a separate PR, but I think these test data generators need to be in the same place; I think this one should probably move into `lib/valid_test_data_generator` and be renamed to match the class

Reply:
Agree, weird place for this. Would be good to move it on a separate PR